src/lib/viewers/controls/model3d/AnimationClipsControl.tsx

```diff
@@ -13,12 +15,47 @@ export type Props = {
     onAnimationClipSelect: () => void;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export default function AnimationClipsControl(props: Props): JSX.Element {
+export const padLeft = (x: number, width: number): string => {
+    return x.length >= width ? x : new Array(width - x.length + 1).join('0') + x;
+};
+
+export const formatDurationStr = (duration: number): string => {
```

Comment:
Do we have something similar to this in the `DurationLabels` component?

Reply:
Yeah you're right, I just copied this over from the original implementation. I'll move it into a `controls/utils.ts` file

Reply:
On further investigation, seems these two serve different functions (although similar). The `DurationLabels` method will strip off the `hours` if it is zero, but this function wants to always display `hh:mm:ss` so unless you have a strong objection, I'll keep this one here